pkg/config/config.go

```diff
@@ -10,10 +10,13 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	nvidia_common "github.com/leptonai/gpud/pkg/config/common"
+	"github.com/leptonai/gpud/pkg/log"
 )
 
 // Config provides gpud configuration data for the server
 type Config struct {
+	AuditLogger log.AuditLogger `json:"-"`
```

Comment:
it's a bit weird that put the logger interface in the structure. we should keep the same style as eventstore.